package.json

```diff
@@ -131,11 +128,16 @@
     "jest-junit": "6.3.0",
     "jest-sonar-reporter": "2.0.0",
     "jest-stare": "1.11.1",
+    "jquery": "3.4.1",
     "jsdoc": "^3.6.2",
+    "jstree": "3.3.8",
     "keytar": "^4.11.0",
     "madge": "3.0.0",
+    "popper.js": "1.15.0",
     "prompt": "1.0.0",
+    "scroll-into-view-if-needed": "2.2.20",
```

Comment:
Do we need to update the `TPSR` list?
@gejohnston, @tucker01, @JamesBauman, @BrandonJenkins14, any thoughts?
https://docs.zowe.org/stable/appendix/tpsr.html#imperative-cli-framework-dependency-attributions

An older version of the same package seems to be part of the API ML.